script/build.ts

```diff
@@ -74,30 +93,34 @@ function packageApp(callback) {
     derefSymlinks: false,
     prune: false, // We'll prune them ourselves below.
     ignore: [
-      '/node_modules/electron($|/)',
-      '/node_modules/electron-packager($|/)',
-      '/\\.git($|/)',
-      '/node_modules/\\.bin($|/)',
+      new RegExp('/node_modules/electron($|/)'),
+      new RegExp('/node_modules/electron-packager($|/)'),
+      new RegExp('/\\.git($|/)'),
+      new RegExp('/node_modules/\\.bin($|/)'),
     ],
     appCopyright: 'Copyright © 2017 GitHub, Inc.',
 
     // macOS
     appBundleId: distInfo.getBundleID(),
     appCategoryType: 'public.app-category.developer-tools',
-    protocols: [
```

Comment:
@joshaber heads up: looks like `protocols` is internal and we should be using `protocol/protocolName` options which they [mash together into the `protocols` array](https://github.com/electron-userland/electron-packager/blob/93160a02b0b190f490e19e4d7e862be5d8a1512a/common.js#L44-L51) - the resulting array is now this:

```json
[
  {
    "schemes": [
      "x-github-desktop-dev-auth"
    ],
    "name": "com.github.GitHubClient"
  },
  {
    "schemes": [
      "x-github-client"
    ],
    "name": "com.github.GitHubClient"
  },
  {
    "schemes": [
      "github-mac"
    ],
    "name": "com.github.GitHubClient"
  }
]
```

which should be semantically the same, but lol computers. Let me know your feels on this.